experience.tex

```diff
@@ -19,20 +19,18 @@ \section{EXPERIENCE}
 \end{bulletlist}
 
 \begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}r}
-    Software Engineer Intern & September 2018 – May 2019\\
-    Wickidcool Software & Milwaukee, WI\\
+    Consultant & July 2021 – October 2021\\
+    Rebentify Technologies & Milwaukee, WI\\
 \end{tabular*}
 \begin{bulletlist}
     \item{
-        Remotely collaborated with other developers using Git to minimize merge conflicts and implement features
-        simultaneously, improving productivity and reduced development timeline.
+        Set up staging and production environments in GCP Cloud Run and firebase to allow for seamless deployments through integrated
+        CI/CD pipelines, ensuring builds were shared across tiers with only configuration changes for security and reproducible behavior.
     }
     \item{
-        Collaborated in development for the front-end using ReactJS and Redux to handle data received from the
-        Node.js back-end of the application, populating it with data from other event services.
+        Integrated existing login system and Login with Google to allow users to quickly login using already existing Google accounts.
```

Comment:
Get rid of "already" - it's implied by existing

Reply:
Good catch! I fixed this in 37fac73, though that includes many more changes to the experience bullets